apps/storybook/src/components/ai-thinking-indicator/index.tsx

```
@@ -0,0 +1,216 @@
+        {...props}
+      >
+        <Indicator size={resolvedSize} />
+        {label && <span>{label}</span>}
+      </div>
+    );
+  }
+);
+
+AIThinkingIndicator.displayName = 'AIThinkingIndicator';
+
+export { AIThinkingIndicator, thinkingIndicatorVariants };
```

ESLint

[error] Fast refresh only works when a file only exports components. Use a new file to share constants or functions between components.

(react-refresh/only-export-components)